integration/kube_test.go

```diff
@@ -328,6 +329,20 @@ func expectedItemsExist(expectedItems []string, actualItems map[string]bool) boo
 	return true
 }
 
+func validateEvents(influxdbClient *influxdb.Client) bool {
+	events, err := getInfluxdbData(influxdbClient, eventsQuery)
```

Comment:
I don't think that is necessary. We just need to verify that events are making it through the pipeline.